packages/layout/src/tree-layout/constants/defaults.js

```diff
@@ -0,0 +1,11 @@
+const DEFAULT_WIDTH = 600;
+const DEFAULT_HEIGHT = 400;
+const DEFAULT_CLASS_NAME = 'muze-layout-container-component';
+const NODE_PREFIX = 'muze-layout-container';
```

Comment:
Keep default class name as "layout-container-component"
Add classprefix from outside...default classprefix will be muze